bgp_session_test.go

```diff
@@ -0,0 +1,124 @@
+package packngo
+
+import (
+	"testing"
+)
+
+var deviceID string
```

Comment:
I think these 3 variables are a bad idea, and will cause chaos in the packngo package. All test-created resources should be local.